### PR TITLE
Svx uodate install

### PR DESCRIPTION
**Pull request by svx**
Fixes: wording

Improves: wording according to best practices

Changes proposed in this pull request:
## 
## 
##

### CODE REVIEW

**Comment by svx** [comment]
would someone mind to review/merge, so that I can open the next/following  PR ?

Thanks !

File: manage/deploying/production/ubuntu_production.rst [hunk 1]
```diff
@@ -49,27 +49,11 @@ Then, install the platform's build kit, nginx, and supervisor:
...
-.. code-block:: console
-
-    tar -xf Plone-5.0-UnifiedInstaller.tgz
-    cd Plone-5.0-UnifiedInstaller
-    sudo ./install.sh
-
-This will install Plone to /usr/local/Plone. There are installer options to put it elsewhere, and the install script will guide you through them. Use ``install.sh --help`` to see all command-line switches.
-
-
+Please take some time and read the cheapter about :doc:`installation </manage/installing/installing/index>`.
```
Comment (ale-rt) [inline manage/deploying/production/ubuntu_production.rst:52]: It should be chapter

File: manage/deploying/production/ubuntu_production.rst [hunk 2]
```diff
@@ -49,27 +49,11 @@ Then, install the platform's build kit, nginx, and supervisor:
...
 
 .. note::
 
-    Note that this is `root` installation. The installer will create special system users to build and run Plone.
+    Note that this is `root` installation `sudo ./install.sh`. The installer will create special system users to build and run Plone.
```
Comment (ale-rt) [inline manage/deploying/production/ubuntu_production.rst:56]: "create special system users"? how many? is it just one, i.e. the sentence should be "create a special system user"?

Comment (polyester) [inline manage/deploying/production/ubuntu_production.rst:56]: no, it creates two. One to do buildout, one to run.

File: manage/deploying/production/ubuntu_production.rst (hunk 1 above)
Comment (polyester) [inline manage/deploying/production/ubuntu_production.rst:52]: aargh. Oversaw that. Good catch

**Pull request merged**